jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -521,10 +522,15 @@ static Fields extractQueryParameters(Request request, Charset charset)
     }
 
     static Fields getParameters(Request request) throws Exception
+    {
+        return getParametersAsync(request).get();
+    }
+
+    static CompletableFuture<Fields> getParametersAsync(Request request)
     {
         Fields queryFields = Request.extractQueryParameters(request);
-        Fields formFields = FormFields.from(request).get();
-        return Fields.combine(queryFields, formFields);
+        CompletableFuture<Fields> contentFields = FormFields.from(request);
+        return contentFields.thenApply(formFields -> Fields.combine(queryFields, formFields));
```

Comment:
Another change that isn't just documentation?  is this what broke the tests in this latest build?